Assignment_05.rb

```diff
@@ -95,22 +138,25 @@ def initialize
   end
 
   def add_one_task(task_attributes)
```

Comment:
@AritraChatterjee-qi Can you please rename this method as `add_task` instead of `add_one_task` ?